pkgs/top-level/all-packages.nix

```diff
@@ -16998,7 +16998,7 @@ with pkgs;
 
   at-spi2-core = callPackage ../development/libraries/at-spi2-core { };
 
-  at-spi2-atk = callPackage ../development/libraries/at-spi2-atk { };
+  at-spi2-atk = at-spi2-core;
```

Comment:
Should we move this to aliases?

Reply:
I want to preserve this in case we decide to split the package in the future.